src/Makefile

```
@@ -0,0 +1,12 @@
+FFLAGS = -ffree-line-length-none
+
+build:
+	gfortran -c poisson.f90 main.f90
+	gfortran poisson.o main.o -o poisson
+	gfortran $(FFLAGS) -c poisson.f90 main.f90
+	gfortran $(FFLAGS) poisson.o main.o -o poisson
```

These two lines have got duplicated somehow

```suggestion
	gfortran $(FFLAGS) -c poisson.f90 main.f90
	gfortran $(FFLAGS) poisson.o main.o -o poisson
```